### PR TITLE
hg-fast-export 160415 (import from head-only)

### PR DESCRIPTION
**Pull request by bfontaine**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --new-formula <formula>` (after doing `brew install <formula>`)?

---

This is a follow-up to #3913; upstream added a tag today :metal: 

https://github.com/Homebrew/homebrew-head-only/pull/215

### CODE REVIEW

**Comment by MikeMcQuaid** [comment]
👍

**Comment by apjanke** [comment]
I'm curious why there was no bottle built for this? I don't see a bottle-disabling directive or unmet platform dependency.

**Comment by bfontaine** [comment]
The only (?) reason this could happens is [this early `return`](https://github.com/Homebrew/brew/blob/31052a924f2d39df69a0e202972b5a9806ff5ac8/Library/Homebrew/dev-cmd/test-bot.rb#L822) but I don’t see why `bottles` would be empty here.

I should add a `bottle :unneeded` anyway.

**Comment by apjanke** [comment]
Merged and added a `bottle :unneeded` as part of an audit fix. Thanks!

File: hg-fast-export.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  head "git://repo.or.cz/fast-export.git"
+
+  def install
+    bin.install "hg-fast-export.py", "hg-fast-export.sh",
+                "hg-reset.py", "hg-reset.sh",
+                "hg2git.py"
+  end
+
+  test do
+    system bin/"hg-reset.sh", "--help"
```
Comment (MikeMcQuaid) [inline hg-fast-export.rb]: Just FYI that I think it's generally nicer to put things in `bin` without their extensions.

Comment (bfontaine) [inline hg-fast-export.rb]: I agree, but upstream doesn’t and [their documentation](https://github.com/frej/fast-export#usage) mentions things with extensions.

Comment (MikeMcQuaid) [inline hg-fast-export.rb]: Ok, cool 👍

**Pull request closed without merging**